src/app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks.ts

```diff
@@ -122,6 +122,7 @@ export function useCurrentAccountNativeSegwitAddressIndexZero() {
  */
 export function useNativeSegwitAccountIndexAddressIndexZero(accountIndex: number) {
   const signer = useNativeSegwitSigner(accountIndex)?.(0);
+  // could it be this?
```

Comment:
Adding this note as the error thrown was `Cannot read property of undefined reading .address" `. 

I see the deprecation warning above this function but I don't understand what it means. 

```

/**
 * @deprecated Use signer.address instead
 */
```

Does that mean I can refactor this to use `signer?.address` instead of `signer?.payment.address`?